File: exercise09/PaintCalculator.java
package exercise09;

import java.util.Scanner;

public class PaintCalculator {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        final int CONVERSION_RATE = 350;
        int length, width, area, tinsOfPaint;
        String output;

        System.out.print("What is the length of the room in feet? ");
        length = scan.nextInt();
        System.out.print("What is the width of the room in feet? ");
        width = scan.nextInt();

        scan.close();

        area = length * width;
        tinsOfPaint = (int)Math.ceil( ((double)area) / CONVERSION_RATE );

        output = String.format("\nYou will need to puchase %d gallons of paint to cover %d square feet.",
            tinsOfPaint, area);

        System.out.println(output);
    }
}

File: 2. Calculations/exercise09/python/paint_calculator.py
# Exercise 09 - Paint Calculator

# import ceiling function from math to round up numbers
from math import ceil

# Declare a constant
CONVERSION_RATE = 350

# Get input from user
length = input('What is the length of the room in feet? ')
width = input('What is the width of the room in feet? ')

# Convert input from string to integer
length = int(length)
width = int(width)

# Perform calculations
area = length * width
tinsOfPaint = int(ceil(float(area) / CONVERSION_RATE))

# Format output
output_text = f'\nYou will need to purchase {tinsOfPaint} gallons of paint' \
              f' to cover {area} square feet.'

print(output_text)
